### ISSUE TITLE
Lightning wallet with remote balance can't receive payments

### ISSUE THREAD

**Issue by salenjak** [mynodebtc/mynode#503]
I can't send any amount (even with fees equal to payment amount) from my Phoenix wallet (also tried with Breez) to RTL wallet. Every payment fails with error message: no route

Was opened a 2m sats channel to one of the top nodes according to BOS Score.
Then was successfuly loop out 1.9m sats to get 1.9m sats on remote balance and 100k on local balance.

Also, I have 2nd channel with 1000 sat on remote, but I still can recieve even payments small as 10 sats or 100 sats.

Does anybody know what can be a reason for not be able to recieve payments although having a channel with plenty of remote capacity?

I have reported the same problem on the RTL's github, but the problem is probably related to my LND
https://github.com/Ride-The-Lightning/RTL/issues/639

**Comment by SatoshiNakamotogmx** [comment]
You might want to double-check the default minimum withdrawal that is on whatever website you are using or what you have set without looking at the error or seeing code that would be the first thing I would check. Isn't there a minimum payment though? Are things really changing that much

**Comment by salenjak** [comment]
I am pasting what was posted on RTL github and closing this issue. 
----------------------------------------------------------------------------
I have opened new channel with OpenNode and after rebalancing i can send recieve everything with normal fees.

The problem was the first channel I opened with SilentBob node. Althoug I have remote balance with this channel for some reason it won't route a payment to my node. Only outgoing transactions.

There is something wrong with this channel because when going in Peers/Channels -----> View Remote Fee I get error message: "Channel Lookup Failed
Error Code
500
Error Message
Unknown Error"

I don't know is the reason SilentBob can't route a payment to me on my LND or their side, but I will close this issue now.
